src/azdo-services.ts

```diff
@@ -0,0 +1,16 @@
+interface CoreMessage {
+  id: string;
+  name: string;
+  locationUrl: string;
+}
+
+export async function getOrganizationUrl(orgName: string) {
+  const resourceAreaUrl = "https://dev.azure.com/_apis/resourceAreas/79134C72-4A58-4B42-976C-04E7115F32BF?accountName=" + orgName;
```

Comment:
Won't this fail on-prem?

Reply:
after testing this in my first commit, it worked as intended online, but it gave me back an incorrect URL on-prem, as you point out. that's why I removed it in that second commit, which instead gets the base URL using the iframe referrer.